setup.cfg

```diff
@@ -3,6 +3,3 @@ project_urls =
     Source=https://github.com/PyCQA/pep8-naming
     Issues=https://github.com/PyCQA/pep8-naming/issues
     Changelog=https://github.com/PyCQA/pep8-naming/blob/master/CHANGELOG.rst
-
-[wheel]
-universal = 1
```

Comment:
I believe this should remain

Reply:
Removing it provides an extra level of protection against incorrectly installing on Python 2.

With this PR, `python3 setup.py bdist_wheel` creates `pep8_naming-0.12.1-py3-none-any.whl`.

But keeping `universal = 1`, it creates `pep8_naming-0.12.1-py2.py3-none-any.whl`, still advertising the `py2` tag.

Renaming the `.whl` to `.zip` for ease of comparison, the contents are identical except for `RECORD` and `WHEEL`:

![image](https://user-images.githubusercontent.com/1324225/150695459-9ae1b1b3-e53d-49cc-bcbb-6c5982bd42d8.png)

`RECORD` just has a different hash:

![image](https://user-images.githubusercontent.com/1324225/150695472-0eba8979-ff15-4a51-adec-aef87ea3375d.png)

But keeping `universal = 1`, `WHEEL` retains metadata for Python 2:

![image](https://user-images.githubusercontent.com/1324225/150695501-92b794e4-c8f8-421c-bac2-d949a8d070f0.png)

And we don't want that Python 2 tag, because it means pip will still install it for Python 2 (bad, it's not going to run):

```console
$ python2 -m pip install -U pep8_naming-0.12.1-py2.py3-none-any.whl
DEPRECATION: Python 2.7 reached the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 is no longer maintained. pip 21.0 will drop support for Python 2.7 in January 2021. More details about Python 2 support in pip can be found at https://pip.pypa.io/en/latest/development/release-process/#python-2-support pip 21.0 will remove support for this functionality.
Processing ./pep8_naming-0.12.1-py2.py3-none-any.whl
Requirement already satisfied, skipping upgrade: flake8-polyfill<2,>=1.0.2 in /usr/local/lib/python2.7/site-packages (from pep8-naming==0.12.1) (1.0.2)
Requirement already satisfied, skipping upgrade: flake8>=3.9.1 in /usr/local/lib/python2.7/site-packages (from pep8-naming==0.12.1) (3.9.2)
Requirement already satisfied, skipping upgrade: enum34; python_version < "3.4" in /usr/local/lib/python2.7/site-packages (from flake8>=3.9.1->pep8-naming==0.12.1) (1.1.6)
Requirement already satisfied, skipping upgrade: typing; python_version < "3.5" in /usr/local/lib/python2.7/site-packages (from flake8>=3.9.1->pep8-naming==0.12.1) (3.7.4)
Requirement already satisfied, skipping upgrade: pycodestyle<2.8.0,>=2.7.0 in /usr/local/lib/python2.7/site-packages (from flake8>=3.9.1->pep8-naming==0.12.1) (2.7.0)
Requirement already satisfied, skipping upgrade: configparser; python_version < "3.2" in /usr/local/lib/python2.7/site-packages (from flake8>=3.9.1->pep8-naming==0.12.1) (3.5.0)
Requirement already satisfied, skipping upgrade: pyflakes<2.4.0,>=2.3.0 in /usr/local/lib/python2.7/site-packages (from flake8>=3.9.1->pep8-naming==0.12.1) (2.3.1)
Requirement already satisfied, skipping upgrade: mccabe<0.7.0,>=0.6.0 in /usr/local/lib/python2.7/site-packages (from flake8>=3.9.1->pep8-naming==0.12.1) (0.6.1)
Requirement already satisfied, skipping upgrade: functools32; python_version < "3.2" in /usr/local/lib/python2.7/site-packages (from flake8>=3.9.1->pep8-naming==0.12.1) (3.2.3.post2)
Requirement already satisfied, skipping upgrade: importlib-metadata; python_version < "3.8" in /usr/local/lib/python2.7/site-packages (from flake8>=3.9.1->pep8-naming==0.12.1) (2.1.1)
Requirement already satisfied, skipping upgrade: contextlib2; python_version < "3" in /usr/local/lib/python2.7/site-packages (from importlib-metadata; python_version < "3.8"->flake8>=3.9.1->pep8-naming==0.12.1) (0.6.0.post1)
Requirement already satisfied, skipping upgrade: zipp>=0.5 in /usr/local/lib/python2.7/site-packages (from importlib-metadata; python_version < "3.8"->flake8>=3.9.1->pep8-naming==0.12.1) (0.5.1)
Requirement already satisfied, skipping upgrade: pathlib2; python_version < "3" in /usr/local/lib/python2.7/site-packages (from importlib-metadata; python_version < "3.8"->flake8>=3.9.1->pep8-naming==0.12.1) (2.3.5)
Requirement already satisfied, skipping upgrade: scandir; python_version < "3.5" in /usr/local/lib/python2.7/site-packages (from pathlib2; python_version < "3"->importlib-metadata; python_version < "3.8"->flake8>=3.9.1->pep8-naming==0.12.1) (1.5)
Requirement already satisfied, skipping upgrade: six in /usr/local/lib/python2.7/site-packages (from pathlib2; python_version < "3"->importlib-metadata; python_version < "3.8"->flake8>=3.9.1->pep8-naming==0.12.1) (1.16.0)
Installing collected packages: pep8-naming
Successfully installed pep8-naming-0.12.1
```

If we ditch the Python 2 tag, it cannot be installed on Python 2 (good):

```console
$ python2 -m pip install -U pep8_naming-0.12.1-py3-none-any.whl
DEPRECATION: Python 2.7 reached the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 is no longer maintained. pip 21.0 will drop support for Python 2.7 in January 2021. More details about Python 2 support in pip can be found at https://pip.pypa.io/en/latest/development/release-process/#python-2-support pip 21.0 will remove support for this functionality.
ERROR: pep8_naming-0.12.1-py3-none-any.whl is not a supported wheel on this platform.
```

And checking the Wheel docs:

> If your project contains no C extensions and is expected to work on both Python 2 and 3, you will want to tell wheel to produce universal wheels by adding this to your `setup.cfg` file:
> 
```
[bdist_wheel]
universal = 1
```

https://wheel.readthedocs.io/en/stable/user_guide.html#building-wheels

> If your project contains no C extensions 

True

> and is expected to work on both Python 2 and 3

False, only expected to work on Python 3.

So we don't need universal wheels.